internal/app/app.go

```diff
@@ -1431,7 +1432,10 @@ func (app *App) repairSlaveOfflineMode(host string, node *mysql.Node, state *Nod
 				app.logger.Errorf("repair: should not turn slave to online until get actual resetup status")
 				return
 			}
-
+			err = node.SetDefaultReplicationSettings(masterNode)
+			if err != nil {
+				app.logger.Errorf("repair: failed to set optimize replication settings on slave %s: %s", host, err)
```

Comment:
Let's make this error message distinguishable from the same message below. It will make it easier to investigate incidents.